docs/0_GettingStarted.md

```diff
@@ -0,0 +1,534 @@
+> program!
+
+- Serial Monitoring is accessible via the shortcut `Ctrl+Shift+M` 
+and clicking the `Serial Monitor` tab.
+Make sure to select the right port and speed (which is `115200` or `9600`) and click
+`Start Monitoring`.
+
+## Network Lab
+
+### Why?
+
+You'll now notice that we also have a Raspberry PI (RPI) on the strandbeest.
+Hence, we need to set-up quite some things to develop on these.
+
+This lab was configured to be able to configure the RPis via SSH, without needing to connect them to a monitor.
+
+To build this lab, you will need:
+
+- a network switch
+- a RPi (which you will configure as a DHCP server and router).
+
+### Instructions
+
+#### Format the micro SD card
+
+In order to work with a Raspberry Pi, you first need to format a micro SD card with the Raspbian OS (or any other OS you are comfortable with).
+
+The steps to do so are the following:
+
+- download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/)
+- choose the version of the RPi your are using (3 model B in our case)
+- choose which OS you want to use (RaspbianOS 64 bit in our case)
+- next, we need to set custom settings in order to enable SSH, click on "Edit Settings"
+- set a hostname, username and password and locale settings according to your region
+- in the "Services" tab, click on "enable SSH" and "Use password authentication"
+- in the "Options" tab, disable "Enable telemetry" (we don't want anyone spying on us 🕵️‍♂️)
+- click on "Save", "Yes" and "Yes" again. Wait a couple of minutes and everything should be good to go 👍.
+
+<br>
+
+Once the micro SD card is flashed, in the `cmdline.txt` file in the boot partition, add the following line: `ip=192.168.1.254`.
+
+Connect both your computer and the RPi to the switch. You should now be able to connect to it via SSH with the aforementioned IP.
+
+> Once connected, don't forget to `sudo apt-get update && sudo apt-get upgrade`.
+
+For the connection to the internet, we will be connecting to the Eduroam network via Wifi. It is a WPA2 network configured with PEAP / MSCHAPv2 authentication (this is import because this creates many complications...).
+
+#### WPA configuration
+
+1. Edit the `/etc/wpa_supplicant/wpa_supplicant.conf` file.
+
+2. Insert the following configuration:
+
+```bash
+ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+update_config=1
+ap_scan=1
+network={
+ssid="eduroam"
+scan_ssid=1
+key_mgmt=WPA-EAP
+eap=PEAP
+identity="${EDUROAM_EMAIL}"
+phase2="auth=MSCHAPV2"
+password="${EDUROAM_PW}"
+anonymous_identity="anonymous@insa-lyon.fr"
+}  
+```
+
+> For the identity and password, you can use your own university email address and password.
+
+It is a good security measure to use environment variables in configuration files when it comes to usernames and passwords so that they don't appear in cleartext on github 😄.
+
+You can configure environment variables like so:
+
+```bash
+export MYVAR='myPassword'
+```
+
+If you want it to be permanent:
+
+```bash
+#TODO
```

Comment:
follow [this](https://unix.stackexchange.com/questions/117467/how-to-permanently-set-environmental-variables).